src/FSharp.Control.TaskSeq.Test/TaskSeq.MaxMin.Tests.fs

```diff
@@ -296,7 +296,7 @@ module SideSeffects =
     }
 
     [<Theory; ClassData(typeof<JustMinMaxBy>)>]
-    let ``TaskSeq-minBy, maxBy with sequence that changes length`` (minMax: MinMax) = task {
+    let ``TaskSeq-minBy, maxBy with sequence that changes length`` (minMax: MinMax) =
```

Comment:
I guess this is ok for now, but keep in mind that there are many bugs with xUnit and async functions, and sometimes it required me to make the whole body to be `task`. Hence you will see it that way, usually (also, it takes less lines, but that is if lesser importance).

Reply:
Agree - it was just the first way I found to shut it up
and I'm def up for having `= task {` on the whole thing in general